### PR TITLE
Add record_info for each test case

### PR DESCRIPTION
**Pull request by mloviska**
- Related ticket: [poo#33160](https://progress.opensuse.org/issues/33160)
- Verification runs:

* [sle-12-SP4-Server-DVD-x86_64-Build0393-btrfs+warnings@64bit](http://dhcp128.suse.cz/tests/6348)
* [sle-15-SP1-Installer-DVD-x86_64-Build43.1-btrfs+warnings@64bit](http://dhcp128.suse.cz/tests/6350#step/partitioning_warnings/1)

### CODE REVIEW

File: lib/partition_setup.pm [hunk 1]
```diff
@@ -66,7 +66,7 @@ sub create_new_partition_table {
     wait_still_screen 2;
     save_screenshot;
     send_key 'ret';
-    if (is_storage_ng) {
+    if (is_storage_ng and !is_opensuse) {
```
[the remark this answers is not in the document]
Comment (mloviska) [reply lib/partition_setup.pm]: It is basically a temporary fix for https://progress.opensuse.org/issues/41357. But as I have noticed that the change has already hit Leap 15.1 this morning. Currently only TW has still issue when creating new partition table https://openqa.opensuse.org/tests/759292#step/partitioning_lvm_thin_provisioning/14

I will revert this line.

File: tests/installation/partitioning_warnings.pm [hunk 2]
```diff
@@ -114,33 +119,45 @@ sub run {
...
     if (is_storage_ng) {
+        ## Test whether /boot is big enough to contain kernel
+        addpart(role => 'OS', size => 40, format => 'ext2', mount => '/boot');
+        record_info('Test: /boot space', "Boot partition size is insufficient to fit kernel");
+        send_key $cmd{accept};
+        process_warning(warning => 'no-space-for-kernel', key => 'alt-o');
+        remove_partition;
+
+        ## Add wrong or small boot partition
         foreach (keys %roles) {
```
Comment (rwx788) [inline tests/installation/partitioning_warnings.pm]: Ok, missed that in VR, this introduces weird behavior, as we test boot/zipl and ofw on x86_64, which is non-sense. I will add this to the follow up ticket we have in the next sprint.

Comment (mloviska) [reply tests/installation/partitioning_warnings.pm]: In fact, that was an intention to check if the partitioner will trigger a warning when there is wrong boot partition.

Comment (rwx788) [reply tests/installation/partitioning_warnings.pm]: You mean that we create /boot/zipl on x86_64 and check if there is still warning about bios_boot? I guess it's a bit of overkill, considering stability and scalability of our GUI tests =(

Comment (mloviska) [reply tests/installation/partitioning_warnings.pm]: Technically, user should be warned when the boot partition does not match the arch. Storage-ng checks the size for all boot partition IDs and for some it checks either ID or mount point or file system, depending on what kind of arch we are running the installer. Also the warning message is still the same, so from the point of stability, we are expecting the same needle to appear for each boot partition ID on tested architecture.

**Pull request merged**